README.md

```diff
@@ -25,83 +92,104 @@ Edit your pom file and add the plugin:
         </configuration>
     </plugin>
     
-    <pluginRepositories>
-        <pluginRepository>
-            <id>gluon-releases</id>
-            <url>https://nexus.gluonhq.com/nexus/content/repositories/releases/</url>
-        </pluginRepository>
-    </pluginRepositories>
-
 The plugin allows some options that can be set in `configuration`, to modify the default settings, and several goals, to build and run the native application.
 
 ### 2. Goals
 
-You can run the regular goals to build and run your project as a regular Java project on the JVM, if you use the `javafx-maven-plugin` plugin:
+#### Desktop
 
-    mvn clean javafx:run
-    
-Once the project is ready, the Client plugin has these main goals:    
+The following goals apply to Linux and Mac OS X.
 
-#### `client:compile`
+To build the native image:
 
-This goal does the AOT compilation. It is a very intensive and lengthy task (several minutes, depending on your project and CPU), so it should be called only when the project is ready and runs fine on a VM.
+    mvn clean client:build
 
-Run:
+To run the native image:
 
-    mvn client:compile
+    mvn client:run
 
-The results will be available at `target/client/gvm`.
+or simply run the native executable found in `target/client`.
 
-#### `client:link`
+#### iOS
 
-When the project is compiled for the target platform, this goal will generate the native executable.
+* Set the target to `ios` (for iOS devices) in the `pom.xml`:
 
-Run:
+```
+<artifactId>client-maven-plugin</artifactId>
+<configuration>
+    <target>ios</target>
+    <mainClass>${mainClassName}</mainClass>
+</configuration>
+```
 
-    mvn client:link
-    
-The results will be available at `target/client/$targetPlatform/$AppName`.
-    
-#### `client:build`
+* Build the native image:
 
-This goal simply combines `client:compile` and `client:link`.
-    
-#### `client:run`
+```
+mvn clean client:build
+```
 
-Runs the executable in the target platform.
+* Run the app on the connected iOS device:
 
-Run:
+```
+mvn client:run
+```
 
-    mvn client:run
-    
-Or run directly the application from command line:
+* Or package and create an IPA file to submit to TestFlight or to the App Store:
 
-    target/client/$targetPlatform/$AppName/$AppName    
-    
-It will create a distributable native application.
+```
+mvn client:package
+```
 
-#### `client:package`
+**Note**: Since all java bytecode is translated to native code, the compilation step can take a long time, and it requires a fair amount of memory.
 
-Create a package of the executable in the target platform.
+**Note**: In order to deploy apps to an iOS device, you need a valid iOS provisioning profile, as explained in the [documentation](https://docs.gluonhq.com/client/#_ios_deployment).
 
-Run:
+#### Android
```

Comment:
This is [not rendered properly](https://github.com/gluonhq/client-maven-plugin/blob/b120bf4e0136b0905a567f80e15e102d147a9c65/README.md).

Reply:
Fixed